awx/main/tasks.py

```diff
@@ -881,6 +880,21 @@ def get_path_to(self, *args):
         '''
         return os.path.abspath(os.path.join(os.path.dirname(__file__), *args))
 
+    def build_execution_environment_params(self, instance):
+        if instance.execution_environment_id is None:
+            self.update_model(instance.pk, execution_environment=instance.resolve_execution_environment())
+
+        image = instance.execution_environment.image
```

Comment:
For whatever reason, some jobs still wind up with a null `execution_environment`, so this needs to be changed to account for that. I'm thinking, this is probably from the project syncs that happen before a job runs.